configs/prompt_muons_example.yaml

```diff
@@ -0,0 +1,58 @@
+chain_name: corsika_snowstorm_prompt_v1_5_1
+n_events_per_run: 100
+n_runs: 10000
+
+dataset_number: 30012
+seed: 3
+
+corsika_module: 'icecube.simprod.modules.Corsika5ComponentGenerator'
+corsika_settings: {
+
+    # things that are pirobably fine like this most of the time
+    'corsikaVersion': '77420',
+    'model': 'SIBYLL',
+    'CutoffType': 'EnergyPerParticle',
+    'oversampling': 1,
+    'trimshower': True,
+    'pnorm': [10., 5., 3., 2., 1.],
+    'pgam': [1., 1., 1., 1., 1.],
+    'UsePipe': True,
+    
+    'ecuts1': 273.,  # hadron min energy (see corsika docs)
+    'ecuts2': 273.,  # muon min energy (see corsika docs)
+    'ecuts3': 273.,  # electron min energy (see corsika docs)
+    'ecuts4': 273.,  # photon min energy (see corsika docs)
+    'CVMFS': '/data/user/pgutjahr/software/CORSIKA/',
+    # things that should be defined by user:
+    'polyplopia': True,
+    'mctree_polyplopia': 'I3MCTree',
+    'eprimarymin': !!float 1e8,
+    'eprimarymax': !!float 1e9,
+    'nuaddi': True,
+}
+muon_propagation_config : {
+    'PROPOSAL_config_file': '/data/user/pgutjahr/software/icetray/v1.5.0/src/PROPOSAL/resources/config_icesim.json',
```

Comment:
is a custom config required? It seems to be the default config by PROPOSAL. Todo: check if this is used and utilize official icetray one